core/src/luigi-config/navigation/queries.js

```diff
@@ -51,6 +51,13 @@ export async function failFastFetch(input, auth, init = {}) {
   }
 }
 
+export async function fetchObservabilityHost(auth, vsPath) {
+  const res = await failFastFetch(config.backendAddress + '/' + vsPath, auth, {
+    method: 'GET',
+  });
+  return (await res.json()).spec.hosts[0];
```

Comment:
do we know if it's always the first port?

Reply:
Please look at the description of this https://github.com/kyma-project/busola/issues/255

Reply:
It's an assumption. We will see if this is not correct